scripts/handle-user-data

```diff
@@ -33,6 +35,10 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
+    mkdir -p /mnt/$PARTNAME/var/lib/origin
+    mkdir -p /var/lib/origin
+    mount --bind /mnt/$PARTNAME/var/lib/origin/ /var/lib/origin/
```

Comment:
Or is there a reason why you did it this way?

BTW, can we not have tests in some form here as well? It would be nice to test some basics with the image itself. At the very least we should build the image in a CI environment.